src/config/logger.ts

```diff
@@ -45,6 +46,22 @@ const errorSerializer = (err) => (!err || !err.stack) ? err : {
 	stack: getFullErrorStack(err)
 };
 
+const hashSerializer = (data: any): string => {
+	if (data === undefined || data == null) {
+		return data;
```

Comment:
this should return `undefined` and not null so it gets removed from the payload when stringifying it.